<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="tj2_romi_i2c" pkg="tj2_romi_i2c" type="tj2_romi_i2c_node.py" output="screen" required="true">
        <param name="sharedmem_path" value="$(find tj2_romi_i2c)/config/sharedmem.json"/>

        <param name="accel_offset_x" value="1.01112"/>
        <param name="accel_offset_y" value="1.00765"/>
        <param name="accel_offset_z" value="0.96534"/>

        <param name="gyro_offset_x" value="2.843"/>
        <param name="gyro_offset_y" value="-6.056"/>
        <param name="gyro_offset_z" value="-1.603"/>
    </node>
</launch>

